lib/iris/tests/test_coding_standards.py

```diff
@@ -80,7 +80,6 @@ class StandardReportWithExclusions(pep8.StandardReport):
     expected_bad_files = [
         '*/iris/std_names.py',
         '*/iris/analysis/_interpolate_private.py',
-        '*/iris/analysis/trajectory.py',
```

Comment:
test_coding_standards objects if a file on the bad list **doesn't** generate any warnings.
This makes us remove sourcefiles from the list once they do pass, so we know if they start failing again
 \- fiendish cunning, eh ?